### PR TITLE
README: update with build instructions

### PR DESCRIPTION
**Pull request by staiyeba**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,37 @@
...
+```
+
+* Profiles:
+
+```
+  conan profile list
+```
+
+
+### Build and run vmbuild
```
Comment (KristianJerpetjon) [inline README.md:32]: This should probably point out to use the toolchain profiles ie apple-clang for mac and clang or gcc for linux

File: README.md [hunk 2]
```diff
@@ -1,3 +1,37 @@
...
+##### View remote and profiles installed
+
+* Remotes:
+```
+  conan remote list
+```
+You should have these:
+
+```
+  includeos: https://api.bintray.com/conan/includeos/includeos [Verify SSL: True]
```
Comment (KristianJerpetjon) [inline README.md]: These arent really needed as your "default" profile is able to build a working vmbuild for you

**Review by KristianJerpetjon — DISMISSED** [review]
You can probably reduce complexity with respect to profiles etc in this one

File: README.md (hunk 2 above)
Comment (staiyeba) [reply README.md]: Removed the texts on remotes.

**Review by mnordsletten — APPROVED** [review]

**Pull request merged**